frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/root/IGitRepositorySelectionProvider.java

```diff
@@ -3,9 +3,11 @@
 import git4idea.repo.GitRepository;
 import io.vavr.collection.List;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
+import org.checkerframework.checker.nullness.qual.Nullable;
 import org.checkerframework.common.value.qual.MinLen;
 
 public interface IGitRepositorySelectionProvider {
+  @Nullable
```

Comment:
Why? :thinking: We can rely on the fact that our plugin is visible only if at least one repository exists, can't we?

Reply:
Changed to `Option` actually; see the updated codebase for rationale (the underlying `com.intellij.ui.CollectionComboBoxModel#getSelected` returns a nullable value :/)